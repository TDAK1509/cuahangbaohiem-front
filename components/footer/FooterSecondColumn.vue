<template>
  <div class="column">
    <h4 class="has-text-weight-bold is-uppercase">Sản phâm</h4>
    <ul class="is-size-7 mt-2">
      <li
        v-for="(nav, index) in navs"
        :key="index"
        class="footer-second-column__list-item mt-1 ml-4"
      >
        <nuxt-link class="footer-second-column__link" :to="nav.to">{{
          nav.text
        }}</nuxt-link>
      </li>
    </ul>
  </div>
</template>

<script>
export default {
  name: "FooterSecondColumn",

  data() {
    return {
      navs: [
        {
          text: "Ô tô",
          to: "/san-pham/o-to"
        },
        {
          text: "Tai nạn 24/24",
          to: "/san-pham/tai-nan"
        },
        {
          text: "Nhà tư nhân",
          to: "/san-pham/nha-tu-nhan"
        },
        {
          text: "Xe máy",
          to: "/san-pham/xe-may"
        }
      ]
    };
  }
};
</script>

<style lang="scss" scoped>
.footer-second-column__link {
  color: #fff;

  &:hover {
    text-decoration: underline;
  }
}

.footer-second-column__list-item {
  list-style-type: disc;
}
</style>
