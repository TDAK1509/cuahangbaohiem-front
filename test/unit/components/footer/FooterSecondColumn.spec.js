import { mount } from "@vue/test-utils";
import FooterSecondColumn from "@/components/footer/FooterSecondColumn.vue";

describe("FooterSecondColumn.vue", () => {
  it("matches snapshot", () => {
    const wrapper = mount(FooterSecondColumn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
