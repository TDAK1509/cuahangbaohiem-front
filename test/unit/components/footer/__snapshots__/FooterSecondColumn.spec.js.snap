// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSecondColumn.vue matches snapshot 1`] = `
<div
  class="column"
>
  <h4
    class="has-text-weight-bold is-uppercase"
  >
    Sản phâm
  </h4>
   
  <ul
    class="is-size-7 mt-2"
  >
    <li
      class="footer-second-column__list-item mt-1 ml-4"
    >
      <nuxt-link
        class="footer-second-column__link"
        to="/san-pham/o-to"
      >
        Ô tô
      </nuxt-link>
    </li>
    <li
      class="footer-second-column__list-item mt-1 ml-4"
    >
      <nuxt-link
        class="footer-second-column__link"
        to="/san-pham/tai-nan"
      >
        Tai nạn 24/24
      </nuxt-link>
    </li>
    <li
      class="footer-second-column__list-item mt-1 ml-4"
    >
      <nuxt-link
        class="footer-second-column__link"
        to="/san-pham/nha-tu-nhan"
      >
        Nhà tư nhân
      </nuxt-link>
    </li>
    <li
      class="footer-second-column__list-item mt-1 ml-4"
    >
      <nuxt-link
        class="footer-second-column__link"
        to="/san-pham/xe-may"
      >
        Xe máy
      </nuxt-link>
    </li>
  </ul>
</div>
`;
